docs/_docs/collections.md

```diff
@@ -87,6 +87,20 @@ For example, if you have `_my_collection/some_subdir/some_doc.md`,
 it will be rendered using Liquid and the Markdown converter of your
 choice and written out to `<dest>/my_collection/some_subdir/some_doc.html`.
 
+If you wish a specific page to be shown when accessing `/my_collection/`,
+simply add `permalink: /my_collection/index.html` to a page.
+To list items from the collection, on that page or any other, you can use:
+
+```liquid
+{% raw %}
```

Comment:
could you put  the`raw`tag _outside_ of the fenced code block (to avoid to render unnecessary white space)?